docs/DEV.md

```diff
@@ -2,157 +2,170 @@
+
+```
+npm run watch
+```
+
+To load the extension manually instead, open (preferably) the [Developer Edition of Firefox](https://www.mozilla.org/firefox/developer/) and load the `.zip` using the following steps:
 
-* Navigate to _about:config_ and set `extensions.legacy.enabled` to `true`. This permits the loading of the embedded Web Extension since new versions of Firefox are becoming restricted to pure Web Extensions only.
 * Navigate to _about:debugging_ in your URL bar
 * Select "Load Temporary Add-on"
-* Find and select the latest xpi file you just built.
+* Find and select the latest zip file you just built.
 
 ## Seeing the add-on in action
 
-To debug installation and loading of the add-on:
+To debug installation and loading of the add-on, check the Browser Console that is automatically opened on start. (Usually accessible using Firefox's top menu at `Tools > Web Developer > Browser Console`).
 
-* Open the Browser Console using Firefox's top menu at `Tools > Web Developer > Browser Console`. This will display Shield (loading/telemetry) and log output from the add-on.
+This will display Shield (loading/telemetry) and log output from the add-on.
 
 See [TESTPLAN.md](./TESTPLAN.md) for more details on how to see this add-on in action and hot it is expected to behave.
 
-## Automated launch of Firefox with add-on installed
-
-`$ npm run firefox` starts Firefox and automatically installs the add-on in a new profile and opens the browser console automatically.
-
-Note: This runs in a recently created profile, and the study variation/branch is overridden by a preference in the FIREFOX_PREFERENCES section of `test/utils.js`.
-
 ## Automated testing
 
-`npm run test` verifies the telemetry payload as expected at Firefox startup and add-on installation in a clean profile, then does **optimistic testing** of the _commonest path_ though the study for a user
+```
+npm run test
+```
+
+Runs tests using the Selenium driver, verifying the telemetry payload at Firefox startup and add-on installation in a clean profile, then does **optimistic testing** of the _commonest path_ though the study for a user:
 
 * prove the notification bar ui opens
 * _clicking on the left-most button presented_.
 * verifying that sent Telemetry is correct.
 
 Code at [/test/functional_test.js](/test/functional_test.js).
 
-Note: The functional tests are using async/await, so make sure you are running Node 7.6+
-
-The functional testing set-up is imported from [https://github.com/mozilla/share-button-study]() which contains plenty of examples of functional tests relevant to Shield study addons.
-
-## Watch
+Note: The study variation/branch during tests is overridden by a preference in the FIREFOX_PREFERENCES section of `test/utils.js`.
 
-You can automatically build recent changes and package them into a `.xpi` by running the following from the top level directory:
+(The functional tests are using async/await, so make sure you are running Node 7.6+)
```

Comment:
Should we just say Node 8.9.0+ to match package.json: `{"engines": { "node": ">=8.9.0" }}`

https://github.com/mozilla/shield-studies-addon-template/pull/55/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R36